manager/controllers/app/blueprint_controller_test.go

```diff
@@ -22,7 +23,7 @@ func deployBlueprint(namespace string, shouldSucceed bool) {
 
 	// Set the correct namespace
 	blueprint.SetNamespace(namespace)
-
+	fmt.Printf("deploy blueprint -  namespace: " + namespace + "\n")
```

Comment:
Consider using format string %s instead of concat

Reply:
Done